### DISCUSSION TITLE
Starting the interactive server resulted in an error

### DISCUSSION THREAD

**Discussion by pomelo-nwu** [alibaba/GraphScope#3413] [Q&A]
Hi, Graphscope team, first of all, thank you for creating such a great project. I'm using Flex/interactive and encountered some issues while following the documentation at https://graphscope.io/docs/flex/interactive/getting_started:

There was an error when initializing gs_interactive, along with a warning.
![image](https://github.com/alibaba/GraphScope/assets/10703060/c2556dfd-58c3-4226-98d8-4a12ae603d66)

It seems that the second step of starting the service also failed.
![image](https://github.com/alibaba/GraphScope/assets/10703060/5622cc20-db86-42cc-8e83-a599950b4228)

My environment is:

```bash
(py39) ➜  interactive uname -aDarwin U-2Q9KNH7F-2210.local 23.1.0 Darwin Kernel Version 23.1.0: Mon Oct  9 21:28:45 PDT 2023; root:xnu-10002.41.9~6/RELEASE_ARM64_T6020 arm64
(py39) ➜  interactive python --versionPython 3.9.18
```

**Comment by zhanglei1949** [comment]
Thank you for providing valuable feedback! It seems to be a problem of `gs_interactive` script on MacOS. We are fixing it now.

**Reply by yecol** [reply]
@zhanglei1949 @pomelo-nwu,
Could you verify that it was fixed?
